README.md

```diff
@@ -84,95 +31,15 @@ Install CMake: <https://www.cmake.org>
      $ cmake -G "Visual Studio 10" ..   # Or whatever generator you want to use cmake --help for a list.
      $ start libevent.sln
 
+See [Documentation/Building#Windows](/Documentation/Building.md#building-on-windows) for more information
 ## CMake (Unix)
```

Comment:
a blank line before headers would be good

Reply:
@ploxiln Review the most recent commit

Reply:
looks good to me, thanks